### ISSUE TITLE
make hardware_events table in windows

### ISSUE THREAD

**Issue by mtmcgrew** [facebook/osquery#2689]
The hardware_events table is only on poxis machines. We should also have it on Windows!

**Comment by PoppySeedPlehzr** [comment]
Yeah yo! This and #2567 both depend on us having the eventing framework being ported to windows.

**Comment by theopolis** [comment]
To be clear, eventing is platform agnostic, you'll only need to write the publishers for Windows.
